test/controllers/events_controller_test.rb

```diff
@@ -3,18 +3,19 @@
 class EventsControllerTest < ActionController::TestCase
   setup do
     log_in_as users(:jeff)
+    @organization = organizations(:csc)
   end
 
   test 'index' do
-    get :index
+    get :index, organization_id: @organization
 
     assert_response :success
-    assert_equal Event.order(date: :desc).page(1), assigns(:events)
+    assert_equal @organization.events.order(date: :desc).page(1), assigns(:events)
   end
 
   test 'index paginates' do
     create_paginated_events
-    get :index
+    get(:index, {'organization_id' => @organization})
```

Comment:
Stylistic suggestion -- I prefer these function calls without the parentheses, without the braces around the hash, and symbols as keys.

```ruby
get :index, organization_id: @organization
```

Feel free to leave the parentheses and curly braces, but I would at least use symbols as keys.